newrelic.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+	/**
+	 * Array of application names.
+	 */
+	app_name: ['YOUR_PROJECT_NAME'],
+	/**
+	 * Your New Relic license key.
+	 */
+	license_key: 'YOUR_NEW_RELIC_LICENSE_KEY',
```

Comment:
Where can we get the key?
Should this Key be secret?

Reply:
It is generated when you create a project (I plan to write a guide for this part). I need to check if it's also passed to frontend or if it's only used on the backend